backend/src/v5/processors/teamspaces/projects/models/containers.js

```diff
@@ -72,4 +75,20 @@ Containers.deleteFavourites = async (username, teamspace, project, favouritesToR
 	return deleteFavourites(username, teamspace, accessibleContainers, favouritesToRemove);
 };
 
+Containers.updateSettings = async (teamspace, container, payload) => {
+	if (payload.defaultView) {
+		await checkViewExists(teamspace, container, payload.defaultView);
```

Comment:
probably better to check this in middleware too - you can run this as a test in yup on those fields - i think 😆